### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cmastr**
view onboarding changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
+## Ensure working directory
+As the products grow, so do the places where documentation lives. Tech writers will end up with more than a few repos that you will work in and contribute to as part of your work. 
+1. From home level in the terminal, type `mkdir d2iq`
+
+## CD into your working directory
+1. Change directory into the new one you just made, or the one you already had:
+`cd d2iq`
+
+## Clone the repo you need
+We will be using the docs site repo for all examples, but this might also be another product repo within D2iQ.
```
Comment (russurquhart1) [inline README.md:38]: May want to add to this to ensure that when they're cloning the repo that SSH is selected.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,68 @@
...
+
+# Initialize the Repo (Docs Repo Only)
+Content that is kept on other team repos would follow their init, if applicable.
+
+## Install node modules
+This must be only done on first time or if a rare site tooling change occurs. Other repos may have other needs, see their Contributing Guide or similar resource for assistance if you will need to build their code.
+`npm install`
+
+## Build the API page sets
+This is necessary anytime a set is changed, or when a new DC/OS version (and there for a new set) is created
```
Comment (russurquhart1) [inline README.md:58]: Is the DC/OS ref ok?

**Review by russurquhart1 — APPROVED** [review]

**Comment by pierrebeitz** [comment]
Quite a bit of that is outdated. Let me have a look after my PTO please!

File: README.md [hunk 3]
```diff
@@ -1,8 +1,68 @@
...
+`code .`
+
+# Initialize the Repo (Docs Repo Only)
+Content that is kept on other team repos would follow their init, if applicable.
+
+## Install node modules
+This must be only done on first time or if a rare site tooling change occurs. Other repos may have other needs, see their Contributing Guide or similar resource for assistance if you will need to build their code.
+`npm install`
+
+## Build the API page sets
```
Comment (samvantran) [inline README.md:57]: Is this part still necessary? Thought Pierre and Daniel simplified this https://github.com/mesosphere/dcos-docs-site#without-docker

File: README.md (hunk 1 above)
Comment (cmastr) [reply README.md:38]: Russ can you help me word this please?

**Pull request merged**